api/v1alpha1/hostedcluster_types.go

```diff
@@ -53,6 +53,9 @@ const (
 	// (performs container image verification).
 	PortierisImageAnnotation = "hypershift.openshift.io/portieris-image"
 
+	//Configure ingress controller with endpoint publishing strategy as Private.
```

Comment:
Missing space before starting the sentence.

Can we elaborate details and context e.g:
```
// Configure ingress controller with endpoint publishing strategy as Private.
This overrides any opinionated strategy set by platform in ReconcileDefaultIngressController.
It's used by IBM cloud for...
NOTE: We'll expose this in the API if the use case gets generalised.
```